cpp/ps0005/ps0005_solution.cpp

```diff
@@ -63,29 +58,31 @@ int main()
         else {
             cout << "(" << num1 << ") / (" << num2 << ") = " << num1 / num2 << "" <<endl;
         }
-    }
+    } // 1/1
 
-    else if (op == '%')
+    else if (op == '%')
     {
-        if (floor(num1) != num1 || floor(num2) != num2)
+        if (floor(num1) != num1 || floor(num2) != num2)
         {
             cout << "cannot execute modulo operation for floating point numbers." <<endl;
         }
-        else if (num2 == 0)
+        else if (num2 == 0)
         {
             cout << "Math Error: cannot perform modulo with 0." <<endl;
         }
         else {
+            // printing the result if all data is valid 1/1
             int result = static_cast<int>(num1) % static_cast<int>(num2);
             cout << "(" << static_cast<int>(num1) << ") % (" << static_cast<int>(num2) << ") = " << result << "" <<endl;
         }
-
-    }
+    } // 1/1
 
     else 
     {
         cout << "Invalid character for operator." <<endl;
-    }
+    } // 1/1
+    // 1/1
 
     return 0;
-}
\ No newline at end of file
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation
- [ ] formatted code to be clear and readable
- [ ] no bugs or warnings while execution
- [ ] commented some phrases for blocks of code
- [ ] main returns non-zero/zero number according to the case
+0.5 bonus mark for your effort ⭐

you can check the [problem solution](https://onlinegdb.com/2mF7i4R8H) for more